src/components/dex/Pair.vue

```diff
@@ -108,6 +107,20 @@ export default {
       return value;
     },
 
+    setMarketData() {
+      this.marketData = _.reduce(this.tickerData, (marketData, market, ticker) => {
+        const close24Hour = this.getClose24Hour(ticker);
+        const change24Hour = this.getChange24Hour(close24Hour, market.open24hr);
+
+        marketData[ticker] = {
+          close24Hour,
+          change24Hour,
+          percentChangeAbsolute: this.getPercentChangeAbsolute(change24Hour, market.open24hr),
```

Comment:
Can `market.open24hr` here be 0? if so this would lead to divide by 0 in `getPercentChangeAsolute`; also can it be undefined?